djangocms_admin_style/sass/partials/_messages.scss

```diff
@@ -1,62 +1,62 @@
 /* Statuses Mixins */
 
 @mixin message-info {
-	@include background-image(linear-gradient( $info-color-light, $info-color 80%, $info-color-dark));
+	background-image: linear-gradient( $info-color-light, $info-color 80%, $info-color-dark);
 	.close {
-		@include box-shadow($info-color 0 0 2px inset);
+		box-shadow: $info-color 0 0 2px inset;
```

Comment:
inset should go first according to the spec - `box-shadow: inset 0 0 2px $info-color`